conf/MA/Morocco01

```diff
@@ -0,0 +1,815 @@
+---	---
```

Comment:
This ain't right 😓 Hopefully it's easy to fix up for you @pcupueran 😅 The tests in this repo should catch problems like this anyway.

Also look out for the filename - should be `/conf/ma/morocco01.yml` (lowercase, with a YML extension)

👀